packages/dds/tree/src/test/feature-libraries/chunked-forest/chunkedForest.spec.ts

```diff
@@ -114,7 +114,7 @@ describe("ChunkedForest", () => {
 			chunk.isShared(),
 			"chunk should be shared after storing as repair data and reinserting",
 		);
-		assert.equal(chunk.referenceCount, 4);
+		assert.equal(chunk.referenceCount, 3);
```

Comment:
This change was done because marks need to be serialized and so during the serialization/deserialization process, the cursor that gets applied to the tree is no longer shared. Is there a convenient way to get around this when applying marks locally?

Reply:
Does using `ITreeCursorSynchronous[]` in the `Revive` mark resolve this problem?

Reply:
It does but the need for serialization is still a problem. The structuredClone implementation we're using doesn't support it so I'm not sure what the best approach would be. 

A few options I thought of:
1. Implement our own structured clone which calls chunkTree on cursors. This doesn't seem like something we'd want to maintain though.
2. Clone using the change encoder instead while working around cursors i.e. serialize and deserialize everything that isn't a cursor.
3. Figure out how to not have to clone marks.

These all seem very involved though so I thought it would be better to deal with it was a separate task if there wasn't an easier way.